src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/parameters/Base64EncodedStringParameterValue/Base64EncodedStringParameterValue.jelly

```diff
@@ -27,6 +27,9 @@ THE SOFTWARE.
         xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form"
         xmlns:i="jelly:fmt" xmlns:p="/lib/hudson/project">
         <f:entry title="${it.name}" description="${it.description}">
+            <div name="parameter" description="${it.description}">
```

Comment:
The *div* element has to present in the jelly file, otherwise the JSON object cannot be parsed properly. The bug fix I mentioned is actually adding the *div* element. The *name* attribute tells Stapler  Request the jelly file serves as one entry if the *parameter* JSON object array. Without the div element, *Base64EncodedString* parameter is outstanding in the JSON and  cannot be picked up as one of the array element.
 Please take a look at line 289 of RebuildAction where the array elements are extracted: https://github.com/jenkinsci/rebuild-plugin/blob/master/src/main/java/com/sonyericsson/rebuild/RebuildAction.java#L289